metaflow/plugins/aws/step_functions/step_functions.py

```diff
@@ -166,6 +166,13 @@ def delete(cls, name):
 
         return schedule_deleted, sfn_deleted
 
+    @classmethod
```

Comment:
same here, rather static method if `cls` is not needed?

Reply:
Here, I do use `cls.get_execution(flow_name, name)`

Reply:
ahh, I see it now! my bad 😓